cuegui/tests/MenuActions_tests.py

```diff
@@ -947,12 +947,13 @@ def test_markAsWaiting(self, yesNoMock):
     def test_dropDepends(self, yesNoMock, frameSearchMock):
         frame_name = 'arbitrary-frame-name'
         frame = opencue.wrappers.frame.Frame(opencue.compiled_proto.job_pb2.Frame(name=frame_name))
+        depend = opencue.wrappers.depend.Depend(opencue.compiled_proto.depend_pb2.Depend())
+
+        frame.getWhatThisDependsOn = lambda: [depend]
         frame.dropDepends = mock.Mock()
 
         self.frame_actions.dropDepends(rpcObjects=[frame])
 
-        frame.dropDepends.assert_called_with(opencue.api.depend_pb2.ANY_TARGET)
```

Comment:
Dropping this line makes the test less effective. Based on the changes you made I think we want to assert that `depend.satisfy` was called. Something like:

```
frame.getWhatThisDependsOn = lambda: [depend]
depend.satisfy = mock.Mock()

self.frame_actions.dropDepends(rpcObjects=[frame])        

depend.satisfy.assert_called()
```

Reply:
@bcipriano - apologies in the delay in resolving this outstanding Cuegui unittest failure. Looking at this again, I do not think that this unittest belongs in Cuegui, because the MenuAction.py `dropDepends` function no longer calls the pycue's `dropDepends` function with a target (which is what what the unittest was asserting with) it instead calls pycue's `depend.satisfy` which already has unittest coverage in Cuegui. My suggestion is to move this `test_dropDepends` to pycue's unittest in `wrappers/frame_test.py/testDropDepends`, where I think it actually belongs because the original test was really a pycue unittest.